artifacts/index.ts

```diff
@@ -1,131 +0,0 @@
-import { BadTokenMock } from "./ts/BadTokenMock";
```

Comment:
I think we need this file for exporting contracts in the `set-protocol-contracts` -package